Graylog2/graylog2-server
# update user session timeout on configuration change

## Description

Update user session token lifetime, when the configuration is changed in the user config overview. 
fixes #15609
## Motivation and Context


When altering a users session timeout policy from "sessions do not time out" to use the global timeout policy, the original cookie is not revoked and thus that user will remain logged in until they decide to logout. Once logged back in the correct timeout is reflected within the cookie.


## How Has This Been Tested?



Steps: 
1. Create user and make sure his sessions never time out 
![image](https://github.com/Graylog2/graylog2-server/assets/98284293/78035a1d-84c9-4b50-8d20-b1f02312b75e)
2. Log in with that user. 
3. Verify in the database that the user session timeout value is -1
4. Update the timeout for the user and verify that the timeout value changed
5. Perform actions as the user and verify that cached session was also updated accordingly
6. Verify that after timeout passed, user is logged out
## Screenshots (if appropriate):

## Types of changes


## Checklist:
